app/res/values/dimens.xml

```diff
@@ -16,6 +16,8 @@
     <dimen name="title_padding">10dp</dimen>
     <dimen name="title_paddingLeft">20dp</dimen>
     <dimen name="tab_gradient_radius">300dp</dimen>
+    <dimen name="detail_view_tab_indicator_height">3dp</dimen>
+    <dimen name="detail_view_tab_padding_horizontal">24dp</dimen>
```

Comment:
Is there a reason for changing these instead of applying the defaults from material design styles ?

Reply:
the default tab indicator height seems a bit too thin:
![image](https://github.com/user-attachments/assets/7fa472ca-a9e7-45be-b202-f20edaf43efd)
And the horizontal padding was already set and I didn't test without, let me do that.

Reply:
In my opinion, it looks better with `24dp` lateral padding (2nd image)
![image](https://github.com/user-attachments/assets/14997179-2965-4897-8ced-5e2c664d7a0b)
What do you think?

Reply:
I agree, thanks for verifying this.